### PR TITLE
Imported claim-admin

### PR DESCRIPTION
**Pull request by rjonczy**
- imported project claim-admin
- added ci workflows for claim-admin

### CODE REVIEW

**Review by VargaElod23 — DISMISSED** [review]
LGTM

File: .circleci/workflows.yml [hunk 1]
```diff
@@ -168,6 +174,42 @@ jobs:
...
+    steps:
+      - *step_install_deps
+      - checkout
+      - *step_setup_remote_docker
+      - *step_prepare_environment
+      - run:
+          name: debug
+          command: |
+            pwd
+            ls
```
[the remark this answers is not in the document]
Comment (rjonczy) [reply .circleci/workflows.yml]: removed

File: legacy/claim-admin/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,97 @@
+def helmDownload() {
```
[the remark this answers is not in the document]
Comment (rjonczy) [reply legacy/claim-admin/Jenkinsfile]: removed

**Review by rattrap — APPROVED** [review]
LGTM

**Comment by rjonczy** [comment]
> Didn't the app work in the services/ dir? It should because it's just a react app

@VargaElod23 and @Malak67 suggested to place it in `/legacy`.

**Pull request merged**